runtime/runtimeUtil.py

```diff
@@ -35,6 +35,8 @@ class BAD_EVENTS(Enum):
     ENTER_IDLE                = "Dawn says enter Idle"
     NEW_IP                    = "Connected to new instance of Dawn"
     DAWN_DISCONNECTED         = "Disconnected to Dawn"
+    TIMESTAMP_DOWN             = "Latency test down, down this stack"
```

Comment:
we probably don't need the second down, so just "Latency test down this the stack" would be clearer. Also try and make the "=" line up with the rest if possible